subprojects/persistent-cache/src/main/java/org/gradle/cache/internal/DefaultPersistentDirectoryCache.java

```diff
@@ -83,6 +83,10 @@ public boolean requiresInitialization(FileLock lock) {
                 return true;
             }
 
+            if (!propertiesFile.exists()) {
+                return true;
```

Comment:
Could you add a logger here like:

```
LOGGER.warn("Invalidating {} as the cache properties file was missing.", DefaultPersistentDirectoryCache.this);
```

Reply:
Sure, will do.

Reply:
@big-guy Just think of another question, did the build fail on a single machine or the specific build fail everywhere? If the former, we can assume there's something unknown happening on that machine.

Reply:
As I understand it, it failed for a particular project, but other builds for different projects on the same machine were OK and the same project on different machines also failed (but these were all "identical" Docker containers).

Reply:
F*. Log statement added.